recipes/libtool/all/conanfile.py

```diff
@@ -162,6 +162,10 @@ def package(self):
             os.rename(os.path.join(binpath, "libtool"),
                       os.path.join(binpath, "libtool.exe"))
 
+        if self.settings.compiler == "Visual Studio" and self.options.shared:
+            os.rename(os.path.join(self.package_folder, "lib", "ltdl.dll.lib"),
+                      os.path.join(self.package_folder, "lib", "ltdl.lib"))
```

Comment:
Just an sanity check, this is the static part, and there's a `bin/ltdl.dll` already in the right spot?

Reply:
Changing the name of an import library, does not affect the name of the expected dll.

I checked it again for this pr:
```
libtool/2.4.6 package(): Packaged 2 '.exe' files: libtool.exe, libtoolize.exe
libtool/2.4.6 package(): Packaged 1 '.dll' file: ltdl-7.dll
libtool/2.4.6 package(): Packaged 8 '.m4' files
libtool/2.4.6 package(): Packaged 2 '.in' files: config-h.in, Makefile.in
libtool/2.4.6 package(): Packaged 7 files
libtool/2.4.6 package(): Packaged 1 '.ac' file: configure.ac
libtool/2.4.6 package(): Packaged 1 '.LIB' file: COPYING.LIB
libtool/2.4.6 package(): Packaged 15 '.c' files
libtool/2.4.6 package(): Packaged 15 '.h' files
libtool/2.4.6 package(): Packaged 1 '.mk' file: ltdl.mk
libtool/2.4.6 package(): Packaged 1 '.am' file: Makefile.am
libtool/2.4.6 package(): Packaged 1 '.guess' file: config.guess
libtool/2.4.6 package(): Packaged 1 '.sub' file: config.sub
libtool/2.4.6 package(): Packaged 1 '.sh' file: ltmain.sh
libtool/2.4.6 package(): Packaged 1 '.lib' file: ltdl.lib
```

So the import library is `ltdl.lib` and the dll is `ltdl-7.dll`.

Reply:
Thanks for confirming I only noticed afterwards the logs are back 🎉